### PR TITLE
8325505: Fix Javadoc ResourceBundle::getString

### PR DESCRIPTION
**Pull request by Thihup**
---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8325505](https://bugs.openjdk.org/browse/JDK-8325505): Fix Javadoc ResourceBundle::getString (**Bug** - P4)

### Reviewers
 * [Justin Lu](https://openjdk.org/census#jlu) (@justin-curtis-lu - Committer)
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17346/head:pull/17346` \
`$ git checkout pull/17346`

Update a local copy of the PR: \
`$ git checkout pull/17346` \
`$ git pull https://git.openjdk.org/jdk.git pull/17346/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17346`

View PR using the GUI difftool: \
`$ git pr show -t 17346`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17346.diff">https://git.openjdk.org/jdk/pull/17346.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17346#issuecomment-1934443556)

### CODE REVIEW

File: src/java.base/share/classes/java/util/ResourceBundle.java [hunk 1]
```diff
@@ -509,7 +509,7 @@ public ResourceBundle() {
      * Calling this method is equivalent to calling
      * {@snippet lang=java :
      *     // @link substring="getObject" target="#getObject(java.lang.String)"
```
Comment (justin-curtis-lu) [inline src/java.base/share/classes/java/util/ResourceBundle.java:511]: Can you do a drive-by update for the comment in the snippet here,

```suggestion
     *     // @link substring="getObject" target="#getObject(java.lang.String)" :
```

and also for the one in `getStringArray(String key)`

**Review by justin-curtis-lu — COMMENTED** [review]
Thanks for the fix, looks correct. In addition to the other comment I left, can you also bump the latter copyright year at the top of the file to 2024.

**Review by naotoj — APPROVED** [review]
LGTM

**Review by justin-curtis-lu — APPROVED** [review]
